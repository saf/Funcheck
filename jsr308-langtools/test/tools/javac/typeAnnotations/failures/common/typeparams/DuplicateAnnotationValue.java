/*
 * @test /nodynamiccopyright/
 * @bug 6843077 6919944
 * @summary check for duplicate annotation values for type parameter
 * @author Mahmood Ali
 * @compile/fail/ref=DuplicateAnnotationValue.out -XDrawDiagnostics -source 1.7 DuplicateAnnotationValue.java
 */
class DuplicateAnnotationValue<@A(value = 2, value = 1) K> {
}

@interface A { int value(); }
