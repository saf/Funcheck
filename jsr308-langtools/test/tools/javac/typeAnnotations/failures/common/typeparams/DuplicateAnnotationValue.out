DuplicateAnnotationValue.java:8:46: compiler.err.duplicate.annotation.member.value: value, A
1 error
